buildSrc/build.gradle.kts

```diff
@@ -153,6 +153,7 @@ fun org.gradle.api.publish.maven.MavenPom.configureForFladle(pluginName: String)
 tasks.withType(Test::class.java).configureEach {
   // Test fixtures are stored in here so we should re-run tests if the test projects change.
   inputs.dir("src/test/resources")
+  maxParallelForks = Runtime.getRuntime().availableProcessors()
```

Comment:
can we make this change separately? How does it affect the performance of the tests?

Reply:
> can we make this change separately?

sure

> How does it affect the performance of the tests?

On my machine, `./gradlew test` took 1m 41s (without parallelism) and 1m 10s (with parallelism)
So it's not something crucial (IMO) but a nice-to-have feature.